File: pratishthaa/date.py
date=input("Enter date: ")
dd,mm,yy=date.split('/')
dd=int(dd)
mm=int(mm)
yy=int(yy)
if(mm==1 or mm==3 or mm==5 or mm==7 or mm==8 or mm==10 or mm==12):
    max1=31
elif(mm==4 or mm==6 or mm==9 or mm==11):
    max1=30
elif(yy%4==0 and yy%100!=0 or yy%400==0):
    max1=29
else:
    max1=28
if(mm<1 or mm>12):
    print("Date is invalid.")
elif(dd<1 or dd>max1):
    print("Date is invalid.")
elif(dd==max1 and mm!=12):
    dd=1
    mm=mm+1
    print("The incremented date: ",dd,mm,yy)
elif(dd==31 and mm==12):
    dd=1
    mm=1
    yy=yy+1
    print("The incremented date: ",dd,mm,yy)
else:
    dd=dd+1
    print("The incremented date: ",dd,mm,yy)

File: Ananya Chandra/date.java
//to predict the date after a number of days from the current date
import java.util.*;
class date
{
public void fu()
{
int S=0;
Scanner sc= new Scanner(System.in);
System.out.println(" enter the day,month,year");
int dd = sc.nextInt();
int mm = sc.nextInt();
int yyyy= sc.nextInt();
System.out.println(" enter the number of days");
int days= sc.nextInt();
int md[]= {0,31,28,31,30,31,30,31,31,30,31,30,31};
if(yyyy%4==0)
md[2]++;
if(dd>md[mm]||mm>12)
{
System.out.println(" invalid input");
}
else
{
S= dd + days;
while(S> md[mm])
{
S=S-md[mm];
mm++;
if(mm==13)
{
mm=1;
yyyy++;
}
}
System.out.println(S+"/"+ mm+"/"+yyyy);
}
}
}
